### PR TITLE
Sineads with test fixes

### PR DESCRIPTION
**Pull request by sineade**
:art:

Want to merge this into Kayt's branch for the design refresh. Just had to adjust the font size of the blue boxes on mobile.

### CODE REVIEW

File: spec/ama_layout/navigation_spec.rb [hunk 1]
```diff
@@ -36,29 +36,32 @@
...
+      end
 
-    it "contains alt text" do
-      expect(subject.items.collect {|i| i.alt }).to eq subject.navigation_items[subject.user.navigation].collect {|i| i["alt"] }
-    end
+      it "contains alt text" do
+        expect(subject.items.collect {|i| i.alt }).to eq subject.navigation_items[user_type].collect {|i| i["alt"] }
+      end
 
     context "member" do
```
Comment (MathieuGilbert) [inline spec/ama_layout/navigation_spec.rb]: I think this line needs to be dropped. Tests failing on syntax error.

**Comment by MathieuGilbert** [comment]
💩

File: spec/ama_layout/navigation_spec.rb [hunk 2]
```diff
@@ -36,29 +36,32 @@
...
-
     it "does not remove nil #navigation_items" do
       expect(subject.items.collect {|i| i.alt }).to include nil
     end
 
-    it "displays links " do
-      expect(subject.items.collect {|i| i.link }).to eq subject.navigation_items[subject.user.navigation].collect {|i| i["link"] }
-    end
+    context "member" do
+      let(:user_type) { "member" }
```
Comment (MathieuGilbert) [inline spec/ama_layout/navigation_spec.rb]: This is already defined on line 40.

File: spec/ama_layout/navigation_spec.rb [hunk 3]
```diff
@@ -36,29 +36,32 @@
   end
 
   describe "#items" do
```
Comment (jbabe) [inline spec/ama_layout/navigation_spec.rb]: @sineade, i think we could either move your changes over into a new branch - or run the following command: 

```bash
git show online-account-refresh:spec/ama_layout/navigation_spec.rb > spec/ama_layout/navigation_spec.rb
```

Comment (jbabe) [inline spec/ama_layout/navigation_spec.rb]: or run:

```bash
git revert 144cf301a3645d329c1157d377656ee95b6ab30c
```

**Pull request closed without merging**